### DISCUSSION TITLE
Select error when run tests

### DISCUSSION THREAD

**Discussion by VadimCeretean4tech** [radix-ui/primitives#1905] [Help]
When I run the tests an error is thrown 

<img width="918" alt="Screenshot 2023-01-24 at 17 36 37" src="https://user-images.githubusercontent.com/117079928/214338573-f85fca40-6ade-4792-a8b6-d094b4bc85da.png">

<img width="1552" alt="Screenshot 2023-01-24 at 17 36 43" src="https://user-images.githubusercontent.com/117079928/214338607-5a25035c-dacb-4ca5-8dad-24d7a225ba00.png">

Something wrong with Select.Content, cause when i removing it, tests work well 

for tests i use jest + @testing-library/react

code example i take from official documentation 
<img width="1375" alt="Screenshot 2023-01-24 at 17 41 29" src="https://user-images.githubusercontent.com/117079928/214339321-2ce28227-36d5-4cc2-98b7-7912ba1ca46d.png">

Any ideas how to fix this?

**Comment by andy-hook** [comment]
Looks like a failure to access `React`, double check you're importing it correctly and that your test environment is properly configured.  If you could post a sandbox environment with a matching issue we can perhaps provide some more guidance.
